### ISSUE TITLE
Reconsider ANSI mappings (especially red and green)

### ISSUE THREAD

**Issue by wilsonminer** [wilsonminer/hyperterm-duotone-dark#2]
The pure duotone color mappings mean error and alert messages don't make sense. Might be better to use a complementary green and red for those colors.

**Comment by simurai** [comment]
Yeah, using red and green for errors and successes would make a bit more sense.

This 👇  _does_ feel a bit confusing:

<img width="202" alt="screen shot 2016-08-05 at 11 57 59 am" src="https://cloud.githubusercontent.com/assets/378023/17433300/2fdd3e36-5b04-11e6-936f-82bf4b220edb.png">
